test/fail.test.js

```diff
@@ -1,14 +1,14 @@
 import test from "ava";
 import nock from "nock";
-import { stub } from "sinon";
+import { spy, stub } from "sinon";
 import fail from "../lib/fail.js";
 import authenticate from "./helpers/mock-gitlab.js";
 
 /* eslint camelcase: ["error", {properties: "never"}] */
 
 test.beforeEach((t) => {
   // Mock logger
-  t.context.log = stub();
+  t.context.log = spy();
```

Comment:
If the [suggestion below](https://github.com/semantic-release/gitlab/pull/680/files#r1495396245) works we can keep this as-is I think.

Reply:
I will undo my changes here. `stub` works, too.